src/content/events/fil-singapore-alongside-token2049-2024.md

```diff
@@ -0,0 +1,25 @@
+---
+title: FIL Singapore alongside TOKEN2049 2024
+created-on: 2024-08-06T10:43:00.000Z
+updated-on: 2024-08-06T10:43:00.000Z
+published-on: 2024-08-06T10:43:00.000Z
+category: supported-sponsored
+location: "Singapore "
+external-link: https://lu.ma/escdw9dx?tk=LRE4fg
+start-date: 2024-09-20T18:00:00.000Z
+end-date: 2024-09-20T22:00:00.000Z
+image:
+  src: /assets/images/fil-singapore-vertical-light.png
+seo:
+  twitter:
+    title: summary
+    card: summary
+  description: >-
+    ​Hosted by MetaPals and supported by Filecoin Foundation, FILSG: Web3
+    Wonderland is the marquee event of Token 2049, bringing together some of the
+    sharpest minds and most innovative projects across Web3 and AI.
+
+
+    ​Enjoy the incredible view of Singapore's skyline and F1 track at the highest food forest in the world - 1-Arden!
```

Comment:
@oliviasigler - The hint in the CMS says" `"To generate meta title and description, copy and paste the entry content into ChatGPT 4.0, then use the prompt: 'Generate a meta title and a meta description under 160 characters for the following content: [paste your content here].'"`

Reply:
Is this supposed to be the main `description` or the SEO description?

Reply:
Join FILSG: Web3 Wonderland at 1-Arden with stunning Singapore views. Discover top Web3 and AI innovations alongside TOKEN2049.